Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,29 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+After Gluster upgrade, it is important that op-version is updated in all peers to avoid any 
+compatibility issues.
+
+###Updating op-version
+
+Once Gluster is upgraded, ensure that the op-version is also updated.
```

Comment:
Again, this is not mandatory. op-version needs to be only bumped up if users are willing to use new features. There shouldn't be any issue if user upgrades the bits and still stick to an older op-version.